website/src/contribute/release-guide.md

```diff
@@ -821,7 +892,8 @@ Start the review-and-vote thread on the dev@ mailing list. Here’s an email tem
     [7] https://github.com/apache/beam-site/pull/...
     [8] https://github.com/apache/beam/pull/...
     [9] https://docs.google.com/spreadsheets/d/1qk-N5vjXvbcEk68GjbkSZTR8AGqyNUM-oLFo_ZXBpJw/edit#gid=...
-
+    [10] https://hub.docker.com/u/apachebeam
```

Comment:
which email do you mean?

Reply:
The line you change is part of an email template (start from line 861). We put links at bottom of the email since they're referenced in above content. Do you want to reference this docker link in email? Otherwise, we can remove it.